clouds/clouds-aws/src/test/java/aws/cloudformation/di/suitetests/AwsEapHostnameTests.java

```diff
@@ -0,0 +1,51 @@
+package aws.cloudformation.di.suitetests;
+
+
+import aws.core.identification.AwsEc2Instance;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import sunstone.api.Parameter;
+import sunstone.api.WithAwsCfTemplate;
+import sunstone.api.inject.Hostname;
+
+import static aws.cloudformation.AwsTestConstants.instanceName;
+import static aws.cloudformation.AwsTestConstants.region;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+/**
+ * Shared resources among other DI tests.
+ * Only one test case, since we don't want Sunstone to create lots of clients.
```

Comment:
This is not clear to me neither.

Reply:
If you inject non-static field, they are injected per test case due to isolation and how JUnit5 framework works. So the idea is to have only one testcase due to performance.

Reply:
I still do not get it :crying_cat_face: 

Are you saying you could do test case for each test method, but you didnt?